python_modules/libraries/dagster-dbt/dagster_dbt/include/dbt_scaffold/definitions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from dagster import Definitions
+
+from dagster_dbt import DbtCli, DbtManifest, dbt_assets
+
+DBT_PROJECT_DIR = Path(__file__).parent.joinpath("..", "jaffle_shop")
+DBT_MANIFEST_PATH = DBT_PROJECT_DIR.joinpath("target", "manifest.json")
+DBT_MANIFEST = DbtManifest.read(path=DBT_MANIFEST_PATH)
+
+
+@dbt_assets(manifest=DBT_MANIFEST)
```

Comment:
Our current dbt quickstarts use `load_assets_from_dbt_project`, which enables an easy local development flow. Is there some way we can preserve that?

Reply:
We could run a subprocess in this file to recreate manifest using `dbt parse`.